src/libxtp/segmentmapper.cc

```diff
@@ -104,11 +100,10 @@ std::vector<double> SegmentMapper<AtomContainer>::getWeights(
 template <class AtomContainer>
 void SegmentMapper<AtomContainer>::ParseFragment(Seginfo& seginfo,
                                                  const tools::Property& frag) {
-  tools::Tokenizer tok_map_atoms(
-      frag.get(_mapatom_xml["atoms"]).template as<std::string>(), " \t\n");
-  std::vector<std::string> map_atoms = tok_map_atoms.ToVector();
-  tools::Tokenizer tok_md_atoms(frag.get("mdatoms").as<std::string>(), " \t\n");
-  std::vector<std::string> md_atoms = tok_md_atoms.ToVector();
+  std::vector<std::string> map_atoms =
+      frag.get(_mapatom_xml["atoms"]).template as<std::vector<std::string>>();
```

Comment:
I don't understand the use of the `template` keyword here :)

Reply:
It is used to indicate that the next < is not a comparison operator but a template parameter 

https://eigen.tuxfamily.org/dox/TopicTemplateKeyword.html